components/dashboard/src/projects/Project.tsx

```diff
@@ -378,17 +378,7 @@ export default function () {
                                                     )}
                                                 </a>
                                                 <span className="flex-grow" />
-                                                <a
-                                                    href={
-                                                        prebuild
-                                                            ? gitpodHostUrl
-                                                                  .withContext(
-                                                                      `open-prebuild/${prebuild.info.id}/${prebuild.info.changeUrl}`,
-                                                                  )
-                                                                  .toString()
-                                                            : gitpodHostUrl.withContext(`${branch.url}`).toString()
-                                                    }
-                                                >
+                                                <a href={gitpodHostUrl.withContext(`${branch.url}`).toString()}>
```

Comment:
We should always start on the branch context here, no matter what prebuild state exists

Reply:
Why? Because we assume everyone will enable incremental workspaces (which results in the best Git state)? :thinking:

Reply:
Ah sorry: Noticed this is the "Branches" view. I agree. :+1: